docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,124 @@
 [[release-highlights-6.3.0]]
-== 6.3.0 release highlights
+== 6.3.0 Release Highlights
 ++++
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
 coming[6.3.0]
 
-See also <<breaking-changes>> and <<release-notes-6.3.0>>.
\ No newline at end of file
+Each release of {kib} brings new features and product improvements. Here are the 
+highlights of the features that were added in 6.3 and how the user experience 
+improved.
+
+Refer to the <<release-notes-6.3.0, {kib} 6.3 Release Notes>> for a list bug 
+fixes and other changes.
+
+[float]
+=== License management and {xpack} code
+
+Starting with version 6.3, all of the free {xpack} features, such as monitoring 
+and tile maps, ship with the default distribution of {kib}.  When you install 
+{kib}, it generates a Basic license with no expiration date.
+
+You can view the status of your license, install a new license, or start a 30-day 
+trial of https://www.elastic.co/subscriptions[Platinum features] in *Management 
+> License Management*. The trial enables you to try out features such as security, 
+machine learning, alerting, graph capabilities, and more.
+
+At the end of the trial period, your license reverts to a Basic license. To 
+continue using the Platinum features, you can purchase a
+https://www.elastic.co/subscriptions[subscription] and update your license.
+
+[role="screenshot"]
+image::images/management_basic_license.png[]
+
+[float]
+=== Autocomplete and simple syntax
+
+{kib}'s experimental autocomplete and simple syntax features can help you create 
+queries. To turn on these features, click *Options* on the right of the query bar
+and opt in.
+
+Once enabled, just start typing in the query bar, and you'll get suggestions for 
+fields, values for keyword fields, and query operators. For information on the 
+simplified, easier-to-use syntax, see <<kuery-query>>.
+
+
+
+[float]
+=== Return to recently viewed content
+
+{kib} now keeps track of your recently viewed items so that you can easily get 
+back to them.  Here's how:
+
+* Your last five recently viewed items are listed at the top of the Home page. 
+A dropdown menu provides access to views further back in history.
+
+* Your recently selected time ranges are listed in the Recent tab in the time 
+picker.
+
+
+[float]
+=== Smaller grid dimensions for dashboards
+
+{kib} 6.3 gives you more control over panel sizes when creating a dashboard.  
+This gives you better control over the layout.  Here are the changes:
+
+* The grid height of the dashboard was decreased from 100 pixels to 20 pixels.
+
+* The number of columns in the grid was increased from 12 pixels to 48 pixels.
+
+
+[float]
+=== Chained Input Controls
+
+The <<controls, Controls visualization>> now enables you to chain dropdown menus. When 
```

Comment:
You can chain the range slider too. Maybe just generically state that we're now allowing for parent / child definition of controls